SolrNet/Impl/SolrBasicServer.cs

```diff
@@ -124,6 +124,12 @@ public ResponseHeader Delete(IEnumerable<string> ids, ISolrQuery q, DeleteParame
             return schemaParser.Parse(schema);
         }
 
+        public LukeResponse GetLuke() {
+            string lukeXml = connection.Get("/admin/luke", null);
+            var lukeResponse = XDocument.Parse(lukeXml);
+            return new LukeResponseParser().Parse(lukeResponse);

```

Comment:
This doesn't compile... the Parse method takes two arguments, could you please fix it?